src/components/accordion/_accordion.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+
+@include govuk-exports("govuk/component/accordion") {
+
+  .govuk-accordion__controls {
+    text-align: right;
+  }
+
+  // This is styled to look like a link not a button
+  .govuk-accordion__expand-all {
+    @include govuk-font($size: 16);
+    display: inline;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: transparent;
+    cursor: pointer;
+  }
+
+  // Hide the body of collapsed sections
+  .govuk-accordion__section[aria-expanded="false"] .govuk-accordion__section-body {
+    display: none;
+  }
+
+  // Border at the bottom of the whole accordtion
+  .govuk-accordion.with-js {
+    border-bottom-width: 1px;
+    border-bottom-style: solid;
+    border-bottom-color: $govuk-border-colour;
+  }
+
+  // Borders between accordion sections
+  .govuk-accordion.with-js .govuk-accordion__section {
+    border-top-width: 1px;
+    border-top-style: solid;
+    border-top-color: $govuk-border-colour;
+  }
+
+  // Section headers have a pointer cursor as an additional affordance
+  .govuk-accordion.with-js .govuk-accordion__section-header {
+    position: relative;
+    padding-top: 14px;
+    padding-bottom: 12px;
+    cursor: pointer;
+  }
+
+  // Headings in section headers have link colours as an additional affodance
+  .govuk-accordion.with-js .govuk-heading-m {
+    margin-top: 0;
+    margin-bottom: 0;
+    color: $govuk-link-colour;
+  }
+
+  // Section headers have a grey background on hover as an additional affodance
+  .govuk-accordion.with-js .govuk-accordion__section-header:hover {
+    background-color: govuk-colour("grey-4");
```

Comment:
@hannalaakso good question. Current practice on the [Service Manual](
https://www.gov.uk/service-manual/helping-people-to-use-your-servic) and at the DVSA is to wrap the `h2` in a button, so that gets focus and displays an orange border.

(The DVSA implementation also wraps the + and - icon in a `<button>` element, so that gets focus too, which seems redundant and not massively accessible)

My implementation currently assigns a `role=button` to the whole header element, on the basis that this better matches the actual clickable area.

However I think adding the same background colour on focus as on hover is a good idea.

Reply:
@joelanman Would you have any thoughts on whether we should be applying the grey background colour when a section heading is in focus (similar to hover state)? Review app [here](https://govuk-frontend-review-pr-958.herokuapp.com/components/accordion).

Reply:
This was resolved by https://github.com/alphagov/govuk-frontend/pull/958#issuecomment-429760943, we're keeping the background colour.